### ISSUE TITLE
Infrastructure for joint W3C-WHATWG publication

### ISSUE THREAD

**Issue by foolip** [whatwg/meta#161]
The [Memorandum of Understanding Between W3C and WHATWG](https://www.w3.org/2019/04/WHATWG-W3C-MOU.html) lays out the process for WHATWG Review Drafts going through the REC process to be endorsed by the W3C, after which the Review Draft is updated to reflect this.

This has happened for the first time now with DOM, see https://dom.spec.whatwg.org/review-drafts/2019-06/. The details of how it was produced are in https://github.com/whatwg/dom/tree/review-drafts-w3c-additions, and updates were also posted in https://github.com/whatwg/dom/pull/767.

Remaining work to make this smoother for future publications:
- [ ] Land https://github.com/tabatkins/bikeshed/pull/1564 so that it need not be applied manually
- [ ] Extract the scripts in https://github.com/whatwg/dom/commit/be3bc514d6db2c158d450bbdcfae23d3b3d9c40f somewhere
- [ ] Update Wattsi to support making similar changes for HTML
- [ ] Track the built output in /review-drafts/ (or more) for each spec in version control
- [ ] Deploy to whatwg.org from that repository

Related issues:
- [Review draft and revamped snapshot work coordination thread](https://github.com/whatwg/meta/issues/92) is where the review draft was originally developed
- [Move ever-growing *.spec.whatwg.org storage off of the VM disk](https://github.com/whatwg/misc-server/issues/107) could be solved together with this issue, but need not be
- [Deploy changes to policy files to whatwg.org](https://github.com/whatwg/sg/issues/110) is similar in that it's publishing built output to whatwg.org

@sideshowbarker @whatwg/editors @whatwg/sg

**Comment by annevk** [comment]
I think we should also extract https://github.com/whatwg/dom/tree/review-drafts-w3c-additions somehow if we want to preserve that. I'd rather not have to keep non-master branches around.

**Comment by foolip** [comment]
Yes, we should transplant that history onto whatever solution we end up with and then delete the branch. I'll add a step.

**Comment by foolip** [comment]
Some stats on /var/www on the whatwg.org server (named "marquee") to inform how what we could/should do for version control:
- A total of ~116k files totaling ~40GB uncompressed and without deduplicating
- Putting all of that into a git repo and running `git repack -a -d -f --depth=250 --window=250` results in a ~440MB repo (The existing whatwg.org repo is already ~340M.)

By comparison, the review-drafts directories have only ~500 files. Putting only those under version control would be no problem at all.

**Comment by TimothyGu** [comment]
Has hosting static files on GitHub Pages been considered? It doesn't sound like it would be too much effort, and we get (essentially) free storage.
